eisbuk-admin/src/components/atoms/AttendanceCard/__tests__/AttendanceCard.test.tsx

```diff
@@ -24,5 +29,9 @@ describe("AttendanceCard", () => {
       screen.getByText("Walter");
       screen.getByText(Category.Competitive);
     });
+    test("should call markAttendance with correct args", () => {
+      userEvent.click(screen.getByText("Mark Walter as present"));
+      expect(mockMarkAttendance).toHaveBeenCalledWith("heisenberg", true);
```

Comment:
Right...so here you're testing that the function has been called with proper args, and that's just it. It has, in fact, been called with those args at some point in the code, but no one guarantees it's return value has been dispatched to the store.
This test should `expect(mockDispatch).toHaveBeenCalledWith(...)`